src/porepy/fracs/fracture_network.py

```diff
@@ -9,16 +9,14 @@
 import porepy as pp
 from porepy.fracs.fracture_network_2d import FractureNetwork2d
 from porepy.fracs.fracture_network_3d import FractureNetwork3d
-from porepy.fracs.line_fracture import LineFracture
-from porepy.fracs.plane_fracture import PlaneFracture
 
 # Custom typings
-Fracture = Union[LineFracture, PlaneFracture]
+FractureList = Union[list[pp.LineFracture], list[pp.PlaneFracture]]
```

Comment:
Is this modification associated with mypy?

Reply:
Yes